kolibri_exercise_perseus_plugin/assets/src/translator.js

```diff
@@ -1,123 +1,130 @@
 import { createTranslator } from 'kolibri.utils.i18n';
 
 const translator = createTranslator('perseusInternalMessages', {
-  "Hint #{ pos }": "Hint #{ pos }",
-  "Show the last step": "Show the last step",
-  "Show the next step": "Show the next step",
-  "Get another hint": "Get another hint",
-  "Steps": "Steps",
-  "Hints": "Hints",
-  "Make sure you select something for every row.": "Make sure you select something for every row.",
+  'Hint #{ pos }': 'Hint #{ pos }',
+  'Show the last step': 'Show the last step',
+  'Show the next step': 'Show the next step',
+  'Get another hint': 'Get another hint',
+  Steps: 'Steps',
+  Hints: 'Hints',
+  'Make sure you select something for every row.': 'Make sure you select something for every row.',
```

Comment:
Isn't this file automatically generated? If so, why did it switch to single quotes instead of double quotes?

Reply:
It is, this is the result of our prettier linting (which gets run automatically on build).

Reply:
Gotcha 👍